src/foundry/foundry.js/searchFilter.d.ts

```diff
@@ -26,28 +56,33 @@ declare class SearchFilter {
 
   /**
    * The CSS selector used to target the tab navigation element
+   * @internal
    */
   protected _inputSelector: string;
 
   /**
    * A reference to the HTML navigation element the tab controller is bound to
+   * @internal
    */
   protected _input: HTMLElement | null;
 
   /**
    * The CSS selector used to target the tab content element
+   * @internal
    */
   protected _contentSelector: string;
 
   /**
    * A reference to the HTML container element of the tab content
+   * @internal
    */
   protected _content: HTMLElement | null;
 
   /**
    * A debounced function which applies the search filtering
+   * @internal
```

Comment:
Why are all of these marked as internal? They are not marked `@private` in the source.

Reply:
Yes, but they are also not marked with `@protected`. That usually means they are still to be considered private, they just haven't been marked yet. If you want, I can clarify this in #dev-support.